### PR TITLE
Fixes #6953 - Fixing N_ calls in User

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -157,11 +157,11 @@ def select_title
   end
 
   def self.anonymous_admin
-    unscoped.find_by_login ANONYMOUS_ADMIN or raise Foreman::Exception.new(N_("Anonymous admin user %s is missing, run foreman-rake db:seed", ANONYMOUS_ADMIN))
```
Comment (domcleal) [inline app/models/user.rb:160]: I think the parentheses are in the wrong place, it should be `Foreman::Exception.new(N_(".."), ANONYMOUS_ADMIN)`.  This passes the English message into the exception class, which generates an error code and _then_ translates it.  If you use % before passing it in, the translation lookup won't work.

Comment (daviddavis) [inline app/models/user.rb:160]: I see, good catch.

Comment (lzap) [inline app/models/user.rb:160]: That is how this is supposed to work, yes. :+1:

**Comment by domcleal** [comment]
Merged as fbea42c for 1.6, thanks @daviddavis!

**Pull request closed without merging**